File: Bot/Handelers/CuffHandeler.js
const LogChannelId = require("./../config").LogChannelId

async function OnCuff(req, res, client){
    try{
        const Cuffed = req.body.Cuffed;
        const CuffedId = req.body.CuffedId;
        const CuffedRole = req.body.CuffedRole;
        const Player = req.body.Player;
        const PlayerId = req.body.PlayerId;
        const PlayerRole = req.body.PlayerRole;

        client.channels.cache.get(LogChannelId).send(`ℹ️ | Hráč ${Player} (${PlayerId}) <${PlayerRole}> zatknul hráče ${Cuffed} (${CuffedId}) <${CuffedRole}>`)

        res.sendStatus(200)
    }
    catch (err){
        res.sendStatus(500)
    }
}


async function OnUnCuff(req, res, client){
    try{
        const Cuffed = req.body.Cuffed;
        const CuffedId = req.body.CuffedId;
        const CuffedRole = req.body.CuffedRole;
        const Player = req.body.Player;
        const PlayerId = req.body.PlayerId;
        const PlayerRole = req.body.PlayerRole;

        client.channels.cache.get(LogChannelId).send(`ℹ️ | Hráč ${Player} (${PlayerId}) <${PlayerRole}> odpoutal hráče ${Cuffed} (${CuffedId}) <${CuffedRole}>`)

        res.sendStatus(200)
    }
    catch (err){
        res.sendStatus(500)
    }
}

module.exports = {OnCuff, OnUnCuff}

File: Plugin/Sitrep Remastered/Handelers/CuffHandeler.cs
using Exiled.Events.EventArgs.Player;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.Http;
using System.Runtime.InteropServices;
using System.Text;
using System.Threading.Tasks;

namespace Sitrep_Remastered.Handelers
{
    internal class CuffHandeler
    {
        public async void OnCuff(HandcuffingEventArgs ev)
        {
            try
            {
                var values = new Dictionary<string, string>
                {
                    {"Cuffed", ev.Target.Nickname },
                    {"CuffedId", ev.Target.UserId },
                    {"CuffedRole", ev.Target.Role.Name },
                    {"Player", ev.Player.Nickname },
                    {"PlayerId", ev.Player.UserId },
                    {"PlayerRole", ev.Player.Role.Name }
                };

                var content = new FormUrlEncodedContent(values);


                await PluginMain.client.PostAsync("http://127.0.0.1:6868/OnCuff", content);
            }
            catch (Exception ex)
            {

            }
        }

        public async void OnUncuff(RemovingHandcuffsEventArgs ev)
        {
            try
            {
                var values = new Dictionary<string, string>
                {
                    {"Cuffed", ev.Target.Nickname },
                    {"CuffedId", ev.Target.UserId },
                    {"CuffedRole", ev.Target.Role.Name },
                    {"Player", ev.Player.Nickname },
                    {"PlayerId", ev.Player.UserId },
                    {"PlayerRole", ev.Player.Role.Name }
                };

                var content = new FormUrlEncodedContent(values);


                await PluginMain.client.PostAsync("http://127.0.0.1:6868/OnUnCuff", content);
            }
            catch (Exception ex)
            {

            }
        }

    }
}
